src/test/codegen/riscv-abi/riscv64-lp64-lp64f-lp64d-abi.rs

```diff
@@ -9,7 +9,7 @@
 #[no_mangle]
 pub extern "C" fn f_void() {}
 
-// CHECK: define zeroext i1 @f_scalar_0(i1 zeroext %a)
+// CHECK: define noundef zeroext i1 @f_scalar_0(i1 noundef zeroext %a)
```

Comment:
Similar here, we should test `--target riscv64-unknown-linux-gnu` and `needs-llvm-components: riscv` here.